dpxdt/server/templates/view_build.html

```diff
@@ -65,17 +65,10 @@
                                         </div>
                                     {% endwith %}
                                     <div class="span2" title="Current status">
-                                        {% if candidate.status == 'good' %}
-                                            <strong>
-                                                {{ candidate.status|title }}
-                                            </strong>
-                                        {% elif candidate.status == 'bad' %}
-                                            <span class="text-error">
-                                                {{ candidate.status|title }}
-                                            </span>
-                                        {% else %}
+                                       {% set class_name = 'text-success' if candidate.status == 'good' else 'text-error' %}
+                                        <span class='{{ class_name }}'>
                                             {{ candidate.status|title }}
```

Comment:
In the case the status isn't `good` or `bad` (so: processing) the old template will just show the text plain. This change makes it so the processing state will now show as an error. So maybe there's a smaller change we can make to this template to make the good state green? I figure it's replacing the `<strong>` tag above with a span that has a class of `text-success`.